.eslintrc.json

```diff
@@ -1,16 +1,115 @@
 {
-    "env": {
-        "browser": true,
-        "commonjs": true,
-        "es2021": true
-    },
-    "extends": "standard",
-    "overrides": [
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jquery": true,
+    "amd":true
+  },
+  "extends": ["plugin:@wordpress/eslint-plugin/recommended"],
+  "globals": {
+    "wp": "readonly"
+  },
+  "rules": {
+    "no-console": "off",
```

Comment:
This is the only config I decided to add comparing with the theme since we're running node and doesn't require the browser

More info: https://eslint.org/docs/latest/rules/no-console